viewer/packages/cad-geometry-loaders/src/CadManager.ts

```diff
@@ -233,8 +233,8 @@ export class CadManager {
    * @param budget The budget to calculate cache size by
    */
   private setCacheSizeForModel(model: CadNode, budget: CadModelBudget) {
-    // This gives cache size of 200 on desktop on default budget
-    const REPOSITORY_CACHE_SIZE_TO_BUDGET_RATIO = 200 / defaultDesktopCadModelBudget.maximumRenderCost;
+    // This gives cache size of 300 on desktop on default budget
+    const REPOSITORY_CACHE_SIZE_TO_BUDGET_RATIO = 300 / defaultDesktopCadModelBudget.maximumRenderCost;
```

Comment:
This change was done after manually testing and observing that 200 was too small for two different mid-size CAD-models. The smaller size did not allow for all sectors in a given camera-view of the model to be cached simultaneously. Increasing it allows Reveal to cache all of the sectors in the view, so that if you e.g. navigate to a position and remove the model before adding it right back in, you will need no additional network requests for sectors to load it.